packages/kmod-6.1-nvidia-r570/nvidia-fabricmanager.service

```
@@ -1,5 +1,7 @@
 [Unit]
 Description=NVIDIA fabric manager service
+Wants=modprobe@ib_umad.service
```

`Wants` is a weak dependency. Would it be better to use `Requires` here?

```suggestion
Requires=modprobe@ib_umad.service
```

---

`Wants` makes sense to me since the module could be built-in to the kernel, and since it's not required by FM for all hardware.

---

I was just getting ready to reply with the same comment, thanks @bcressey!